stable/insights-admission/templates/_specs.yaml

```
@@ -24,4 +24,30 @@
 - name: NO_PROXY
   value: {{ .Values.global.proxy.no_proxy }}
 {{- end }}
-{{- end }}
\ No newline at end of file
+{{- end }}
+
+{{ define "ssl-cert-file-volume-spec" }}
+{{- if and .Values.global.sslCertFile.secretName .Values.global.sslCertFile.secretKey }}
+- name: sslcertfile
+  secret:
+    secretName: {{ .Values.global.sslCertFile.secretName }}
+    items:
+    - key: {{ .Values.global.sslCertFile.secretKey }}
+      path: ca.crt
+{{- end }}
+{{- end }}
+
+{{ define "ssl-cert-file-volumemount-spec" }}
+{{- if and .Values.global.sslCertFile.secretName .Values.global.sslCertFile.secretKey }}
+- name: sslcertfile
+  mountPath: /ssl-cert-file
+{{- end }}
+{{- end }}
+
+{{ define "ssl-cert-file-env-spec" }}
+{{- if and .Values.global.sslCertFile.secretName .Values.global.sslCertFile.secretKey }}
+- name: SSL_CERT_FILE
+  value: /ssl-cert-file/ca.crt
+  {{- end }}
+  {{- end }}
```

minor indentation issue
```suggestion
{{- end }}
{{- end }}
```